tox.ini

```diff
@@ -28,9 +28,10 @@ description =
 
 deps =
 # The following indicates which extras_require from setup.cfg will be installed
+    scipy
```

Comment:
Why is this here when its part of the setup.cfg?

Reply:
Sorry, this was a local virtualenv issue